spec/helpers/geoblacklight_helper_spec.rb

```diff
@@ -357,4 +357,29 @@
       end
     end
   end
+
+  describe "#viewer_container" do
+    let(:document) { SolrDocument.new(document_attributes) }
+    let(:references_field) { Settings.FIELDS.REFERENCES }
+    let(:rights_field) { Settings.FIELDS.RIGHTS }
+
+    context "with pmtiles layer" do
+      let(:ol_viewer) { double }
+      let(:document_attributes) do
+        {
+          references_field => {
+            "https://github.com/protomaps/PMTiles" => "https://test-data.cloudgbl.org/tufts-cambridgegrid100-04.pmtiles"
```

Comment:
This all looks and works great for me. I'm gonna make one little change here to swap this spec PMTile endpoint (was brittle) for your new fixture PMTile endpoint (more stable) — even though this won't have any impact on running the spec.